scheduler/cups-driverd.cxx

```diff
@@ -113,11 +115,17 @@ typedef union				/**** TAR record format ****/
   }	header;
 } tar_rec_t;
 
+struct args {
```

Comment:
Like @zdohnal says, see the coding guidelines for the naming conventions and coding style we use

Wrap the struct in a typedef.  "args" isn't a meaningful name for a structure, and there are no comments to tell me what the structure or members are used for.